### PR TITLE
Cleanup dependencies

### PR DESCRIPTION
**Pull request by rouge8**
This patch removes some unused requirements and unpins the versions of the other core dependencies to avoid conflicting with code in other applications.

I suspect the remaining dependencies could be made more flexible, but unpinning requests was my main concern.

### CODE REVIEW

**Comment by aliabbasrizvi** [comment]
Thanks @rouge8 for the change ... looking into this change now.

File: requirements/core.txt [hunk 1]
```diff
@@ -1,7 +1,3 @@
-jsonschema==2.5.1
-mmh3==2.3.1
-pbr==1.8.1
-PyYAML==3.11
-requests[security]==2.9.1
-sh==1.11
-six==1.10.0
+jsonschema>=2.5.1
+mmh3>=2.3.1
```
Comment (aliabbasrizvi) [inline requirements/core.txt]: @rouge8 I would like to keep this at 2.3.1. Other 2 are fine.

**Comment by aliabbasrizvi** [comment]
Thanks for the cleanup @rouge8 ... just the one comment.

**Comment by rouge8** [comment]
@aliabbasrizvi fixed

**Comment by rouge8** [comment]
Thanks! Will this make it into a release fairly soon?

**Comment by aliabbasrizvi** [comment]
@rouge8 yes !! we are planning to do a release later this week.

**Comment by aliabbasrizvi** [comment]
@rouge8 version `0.1.2` with this fix is live now.

**Pull request merged**